vixos/template_nix.py

```diff
@@ -24,7 +28,8 @@ def generate_nix_user(package: str, executable: str) -> str:
   imports = [
     <nixpkgs/nixos/modules/virtualisation/qemu-vm.nix>
     ./managed.nix
-    ../local.nix
+    ./local.nix
+    ./global.nix
```

Comment:
Do we need three configuration files (local, managed and default)?

Also if I understand the idea correctly, the template is not `nix_user` anymore, but `nix_default`. Maybe change the function name?

Reply:
Done.